util/topgen/intermodule.py

```diff
@@ -534,7 +534,8 @@ def find_otherside_modules(topcfg: OrderedDict, m,
         ('main', 'tl_corei'): ('rv_core_ibex', 'tl_i'),
         ('main', 'tl_cored'): ('rv_core_ibex', 'tl_d'),
         ('main', 'tl_dm_sba'): ('dm_top', 'tl_h'),
-        ('main', 'tl_debug_mem'): ('dm_top', 'tl_d')
+        ('main', 'tl_debug_mem'): ('dm_top', 'tl_d'),
+        ('peri', 'tl_ast_wrapper'): ('ast', 'tl')
```

Comment:
Do we need this ? I thought this one is only used for `inter_signal.top` not the `ext`.. maybe I don't understand what I coded :(

Reply:
well when i didn't find this i kept getting an error that the find signal didn't work :)
I think we might not need it in design, but the DV guys might need it.

Reply:
I assume all IP is inside top_earlgrey, but ast isn't. Will make a PR to update it once this is merged.

Reply:
perhaps comment out line 167 at https://github.com/lowRISC/opentitan/blob/master/hw/ip/tlul/generic_dv/xbar_tests.hjson, to let CI pass.
> // tests: ["xbar_{name}_sanity"]

Reply:
yeah i'm giving a random try to fix things right now. If i can't , that's what i'm going to do :)